CHANGES.rst

```diff
@@ -58,6 +58,8 @@ outlier_detection
 
 - Replace uses of ``datetime.utcnow`` (deprecated in python 3.12) [#8051]
 
+- Moved build configuration from ``setup.cfg`` to ``pyproject.toml`` to support PEP621 [#6847]
```

Comment:
Why is this entry under `outlier_detection`?

Reply:
moved to `general`